### PR TITLE
[kubevirt][ssp][common-templates] Add go functests lane to common-templates

### PR DESCRIPTION
**Pull request by vatsalparekh**
Add a test lane for golang functional tests

Signed-off-by: Vatsal Parekh <vparekh@redhat.com>

### CODE REVIEW

File: ci-operator/config/kubevirt/common-templates/kubevirt-common-templates-master.yaml [hunk 1]
```diff
@@ -34,6 +34,27 @@ tests:
           cpu: 200m
           memory: 400Mi
     workflow: ipi-gcp
+- as: go-funtests
```
Comment (akrejcir) [inline ci-operator/config/kubevirt/common-templates/kubevirt-common-templates-master.yaml:37]: Can you keep the naming convention similar to other tests?
For example: `e2e-azure-go-tests`.

**Review by akrejcir — CHANGES_REQUESTED** [review]
Can you prefix the commit message and the PR name with `kubevirt: ssp:`?
It will help to identify that this commit and PR belong to our project.
This repository contains test configuration for many projects, so a commit name `Add go functests lane` is too generic, and it could apply to any project.

File: ci-operator/config/kubevirt/common-templates/kubevirt-common-templates-master.yaml (hunk 1 above)
Comment (vatsalparekh) [reply ci-operator/config/kubevirt/common-templates/kubevirt-common-templates-master.yaml]: corrected

**Pull request merged**